src/util/songs.js

```diff
@@ -0,0 +1,11 @@
+import { pick, values } from 'ramda'
+
+const cleanSearchTerm = term => term.toLowerCase().replace(/ /g,'')
+
+export const songContainsSearchTerm = searchTerm => song => {
```

Comment:
In general, whats the benefit to this over something like ```
songContainsSearchTerm = (searchTerm,song) => {
```

Reply:
just preference. maybe comes from a lot of exposure to FP, but I think it makes the semantics of filtering a little more straightforward, and gives us a little room (which we probably won't actually need) to define
```js
const isFromMovingPictures = songContainsSearchTerm('Moving Pictures')
const getMovingPicturesSongs = filter(isFromMovingPictures)
```
probably a more practical application of this concept, if we needed this functionality:
```js
const hasFewerVotesThan = num => song => song.votes < num
const isUnpopular = hasFewerVotesThan(5)
const isPopular = compose(not, hasFewerVotesThan(30))
const isMeh = compose(not, either(isUnpopular, isPopular))
const getUnpopularSongs = filter(isUnpopular)
const getPopularSongs = filter(isPopular)
```

Reply:
also, I can change this implementation for the filtering if you don't like it.

Reply:
I like it! I think FP is just new for me so takes a second longer to read, but it's good practice for me, so let's lean into it.